cluster/eksctl/cluster.yaml

```diff
@@ -1,44 +1,38 @@
 apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
-
```

Comment:
I use yq to simplify the updates of that files but it trimmed the whitespace. I believe this doesn't affect the readability too much, but let me know what you think.